src/main/webapp/query-control.js

```diff
@@ -290,6 +296,18 @@ function replaceValueIfEmpty(dataListId) {
   }
 }
 
+// Show the div that allows for AmCharts map title editing
+function showEditTitle() {
+  document.getElementById('edit-title').style.display = 'inline';
```

Comment:
you should pre-populate the textbox with the current title

Reply:
Good idea! Done.